src/ui/pages/group/dialogs/AddTransactionDialog.js

```diff
@@ -10,27 +10,25 @@ const AddTransactionDialog = ({ ...props }) => {
     };
 
     return (
-        <div>
```

Comment:
Basically turns out the wrapping div is not needed, so I removed it from all dialogs